app/db/models/add-address.model.js

```diff
@@ -0,0 +1,15 @@
+const Sequelize = require('sequelize');
+const database = require('../');
+
+const Address = database.define('Address', {
```

Comment:
Bikeshed moment? But `Address` makes me think of a mailing address. Maybe we should make this more specific.

Reply:
I agree with gmem, a different model name would be appreciated.